.github/workflows/ci.yml

```diff
@@ -113,6 +113,28 @@ jobs:
         python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
         python builder.pyz build -p ${{ env.PACKAGE_NAME }}
 
+  windows-vc16:
+    runs-on: windows-2019 # windows-2019 is last env with Visual Studio 2015 (v14.0)
+    strategy:
+      matrix:
+        arch: [x86, x64]
+    steps:
+    - name: Build ${{ env.PACKAGE_NAME }} + consumers
+      run: |
+        python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
+        python builder.pyz build -p ${{ env.PACKAGE_NAME }} --target windows-${{ matrix.arch }} --compiler msvc-16
+
+  windows-vc15:
+    runs-on: windows-2019 # windows-2019 is last env with Visual Studio 2015 (v14.0)
+    strategy:
+      matrix:
+        arch: [x86, x64]
+    steps:
+    - name: Build ${{ env.PACKAGE_NAME }} + consumers
+      run: |
+        python -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
+        python builder.pyz build -p ${{ env.PACKAGE_NAME }} --target windows-${{ matrix.arch }} --compiler msvc-15
+
```

Comment:
Yeah, it does.
> For windows 2019 with MSVC 15, _addcarry_u32 and _addcarry_u64 has a weird bug that if the first number is 0, the returned carry out overflow flag will ALWAYS be set, which makes me Worried, as it will affect all our downstream and who knows what else weird bug is out there.